components/event-chart/src/event-chart.ts

```diff
@@ -260,6 +264,10 @@ export class DtEventChart<T> implements AfterContentInit, OnInit, OnDestroy {
     // tslint:disable-next-line: no-any
     @Inject(DOCUMENT) private _document: any,
     private _platform: Platform,
+    @Optional()
+    @Inject(DT_UI_TEST_CONFIG)
+    private _config?: DtUiTestConfiguration,
+    private _elementRef?: ElementRef<HTMLElement>,
```

Comment:
please move before @Optional and add deprecation notice to remove being optional